core/chains/evm/logpoller/integration_test.go

```diff
@@ -23,6 +23,12 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+func logRuntime(t *testing.T, f func()) {
+	s := time.Now()
+	f()
+	t.Log("runtime", time.Since(s))
+}
```

Comment:
style nit, but maybe reads a little easier :shrug: 
Could simplify here a bit:
```suggestion
func logRuntime(t *testing.T) func() {
	s := time.Now()
	return func() { t.Log("runtime", time.Since(s))	}
}
```
and then call it more directly, like:
```go
func() {
	defer logRuntime(t)
	...
}()
```